libc/cmake/caches/armv6m-none-eabi.cmake

```diff
@@ -2,7 +2,7 @@ set(CMAKE_SYSTEM_PROCESSOR arm CACHE STRING "")
 set(RUNTIMES_TARGET_TRIPLE "armv6m-none-eabi" CACHE STRING "")
 
 foreach(lang C;CXX;ASM)
-    set(CMAKE_${lang}_FLAGS "-march=armv6m -mcpu=cortex-m0plus -mfloat-abi=soft -Wno-atomic-alignment \"-Dvfprintf(stream, format, vlist)=vprintf(format, vlist)\" \"-Dfprintf(stream, format, ...)=printf(format)\" \"-Dfputs(string, stream)=puts(string)\" -D_LIBCPP_PRINT=1" CACHE STRING "")
+    set(CMAKE_${lang}_FLAGS "-march=armv6m -mcpu=cortex-m0plus -mfloat-abi=soft -Wno-atomic-alignment" CACHE STRING "")
```

Comment:
Bikeshedding a bit, but why do we need `-Wno-atomic-alignment`?

Reply:
That's because we use atomics for some internal API and there's no support unaligned accesses on Cortex-M0, but it's not a problem in practice because we don't do any unaligned accesses. We expect this will eventually get resolved by switching to TLS but we haven't had time to implement it yet.